test/unit/adapters/http.js

```diff
@@ -428,6 +428,21 @@ describe('supports http with nodejs', function () {
     });
   });
 
+  it('should display error while parsing params', function (done) {
+    server = http.createServer(function (req, res) {
```

Comment:
Are we not using a linter, there are unused variables `req` and `res` ?

Reply:
Yeah we are using a linter but it doesn't have a problem with unused variables and I didn't see it.

Reply:
That is strange, might be the linter have relaxed the rule for unused variables.

Reply:
Do you suggest any other changes?

Reply:
Rest looks good to me.

Reply:
k